package-lock.json

```diff
@@ -1,8 +1,3599 @@
 {
   "name": "safespace",
   "version": "0.1.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
```

Comment:
This may break the app. Was this the branch that you installed the dependency on?